applications/flash_attention_v2/kernel/xe_flash_attn_gemm.hpp

```diff
@@ -317,28 +333,38 @@ class GemmUniversalAttention {
       // MAIN LOOP: loop over K and V, perform fused attention + online softmax
       for (int nblock = 0; nblock < nblock_limit - static_cast<int>(CausalMask); nblock++) {
         barrier_arrive(barrier_scope);
-        // 1) Load K (performed inside mmaQK)
+
+        bool is_KV_cache = (nblock < nblock_cache);
```

Comment:
```suggestion
        bool is_KV_cache = nblock < nblock_cache;
```
redundant parenthesis